docs/document_store_schema.md

```diff
@@ -0,0 +1,384 @@
+    }
+    is_active: bool
+    is_council_president: bool
+    most_recent_seat: {
+        id: str
+        name: str
+        electoral_area: str
+        map_file_id: str
+        map_uri: str
+    }
+    most_recent_chair_body: {
+        id: str
+        name: str
+    }
+    terms_serving_in_current_seat_role: int
+    terms_serving_in_current_committee_chair_role: int
+    external_source_id: optional[any]
+    updated: datetime
+    created: datetime
+}
+```
+
+### Examples
+
+
+## Body
+A body, also known as committee, is a subset of city council members that stands for a certain topic/purpose.
+An example would be the Seattle "Governance and Education" committee which consists of 6 of the 9 city council members.
+This can however include general categories such as "Debate", or "Press Conference", etc.
+
+### Schema
+```
+body_id: {
+    name: str
+    tag: str
+    description: optional[str]
+    start_date: datetime
+    end_date: optional[datetime]
+    is_active: bool
+    chair_person_id: str
+    external_source_id: optional[any]
+    updated: datetime
+    created: datetime
+}
+```
+
+### Examples
+
+
+## File
+A collection to coordinate file details between a database and file store.
+
+### Schema
+```
+file_id: {
+    uri: str
+    filename: str
+    description: optional[str]
+    content_type: optional[str]
+    created: datetime
+}
+```
+
+### Examples
+
+
+## Transcript
+The primary transcript for an event.
+
+### Schema
+```
+transcript_id: {
+    event_id: str
+    file_id: str
+    confidence: float
+    created: datetime
+}
+```
+
+### Examples
+
+
+## Seat
+A seat is an electable office on the City Council or Executive position.
+
+### Schema
+```
+seat_id: {
+    name: str
+    electoral_area: str
+    electoral_type: str
+    map_file_id: str
+    map_uri: str
+    created: datetime
+}
+```
+
+### Examples
+
+### Notes
+These are managed from a pre-populate database script that should be ran prior to any other "streaming" data entering
+the system.
+
+
+## Role
+A role is a person's job for a period of time in the city council. A person can (and should) have multiple roles.
+For example: a person has two terms as city council member for D4 then a term as city council member for all-city.
+Roles can also be tied to committee chairs. For example: a council member spends a term on the transportation committee
+and then spends a term on the finance committee.
+
+### Schema
+```
+role_id: {
+    person: {
+        id: str
+        name: str
+    }
+    title: str
+    body: {
+        id: str
+        name: str
+    }
+    start_date: datetime
+    end_date: optional[datetime]
+    seat_id: str
+    external_source_id: optional[any]
+    created: datetime
+}
+```
+
+### Examples
+
+
+## Minutes Item
+A minutes item is anything found on the agenda / minutes. This can be a matter but it can be a presentation or budget
+file, etc
+
+### Schema
+```
+minutes_item_id: {
+    minutes_item_id: str
+    name: str
+    title: str
```

Comment:
Does a minutes item that isn't tied to matter have a `title`? If it doesn't have `title`, can we remove line 222?
If the minutes item is tied to a matter, it's title would be found in `matter.title`.

Reply:
Good question.... I believe it does, in a sense that it is common right now for certain minutes items for be "named" something like 'Jun 16, 2019' but the official title is 'Approval of Minutes'.

Reply:
Check out this [this 4/20 meeting](http://seattle.legistar.com/MeetingDetail.aspx?ID=775475&GUID=B1AC590C-3293-436C-8FEF-7EFABD82896C&Options=info|&Search=). I think the kind of minute items you mentioned are actually related to matters. For example, the Min 277 minutes item, I think, in our DB, its `matter.name` would `Min 277` and `matter.title` would be `April 13, 2020`. As I understand it the non-matter related minute items are the ones with uppercase titles like CALL TO ORDER, ROLL CALL, PRESENTATIONS, etc.  In our DB, I'm proposing these minutes items have a `name` only. For example, the CALL TO ORDER minutes item's `name` would be CALL TO ORDER.

Reply:
Gotcha. Agree!